Can't run `pnpm dev:playground` Failed to resolve entry for package "@spotlightjs/sidecar".

Steps to repro (at sha: https://github.com/getsentry/spotlight/commit/2e2d8ffee8658089fad1df6e8a225734f59f23e0):

```
git clone https://github.com/getsentry/spotlight.git

volta install node pnpm
pnpm install
pnpm dev:playground
```

Error:

```
bruno@DHPKTVF22X spotlight % pnpm dev:playground

> @ dev:playground /Users/bruno/git/spotlight
> pnpm --filter astro-playground dev


> @spotlightjs/astro-playground@0.0.0 dev /Users/bruno/git/spotlight/demos/astro-playground
> astro dev

Failed to resolve entry for package "@spotlightjs/sidecar". The package may have incorrect main/module/exports specified in its package.json.
11:18:24 p.m. [vite] Error when evaluating SSR module /Users/bruno/git/spotlight/demos/astro-playground/astro.config.mjs: failed to import "/@fs/Users/bruno/git/spotlight/packages/astro/src/index.ts"
|- Error: Failed to resolve entry for package "@spotlightjs/sidecar". The package may have incorrect main/module/exports specified in its package.json.
    at packageEntryFailure (file:///Users/bruno/git/spotlight/node_modules/.pnpm/vite@4.5.0/node_modules/vite/dist/node/chunks/dep-bb8a8339.js:28725:11)
    at resolvePackageEntry (file:///Users/bruno/git/spotlight/node_modules/.pnpm/vite@4.5.0/node_modules/vite/dist/node/chunks/dep-bb8a8339.js:28722:5)
    at tryNodeResolve (file:///Users/bruno/git/spotlight/node_modules/.pnpm/vite@4.5.0/node_modules/vite/dist/node/chunks/dep-bb8a8339.js:28453:20)
    at Context.resolveId (file:///Users/bruno/git/spotlight/node_modules/.pnpm/vite@4.5.0/node_modules/vite/dist/node/chunks/dep-bb8a8339.js:28212:28)
    at Object.resolveId (file:///Users/bruno/git/spotlight/node_modules/.pnpm/vite@4.5.0/node_modules/vite/dist/node/chunks/dep-bb8a8339.js:44276:64)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async TransformContext.resolve (file:///Users/bruno/git/spotlight/node_modules/.pnpm/vite@4.5.0/node_modules/vite/dist/node/chunks/dep-bb8a8339.js:43992:23)
    at async normalizeUrl (file:///Users/bruno/git/spotlight/node_modules/.pnpm/vite@4.5.0/node_modules/vite/dist/node/chunks/dep-bb8a8339.js:41836:34)
    at async file:///Users/bruno/git/spotlight/node_modules/.pnpm/vite@4.5.0/node_modules/vite/dist/node/chunks/dep-bb8a8339.js:41998:47
    at async Promise.all (index 7)

[astro] Unable to load your Astro config

 error   Failed to resolve entry for package "@spotlightjs/sidecar". The package may have incorrect main/module/exports specified in its package.json.
  File:
    /Users/bruno/git/spotlight/packages/astro/src/index.ts
  Stacktrace:
Error: Failed to resolve entry for package "@spotlightjs/sidecar". The package may have incorrect main/module/exports specified in its package.json.
    at packageEntryFailure (file:///Users/bruno/git/spotlight/node_modules/.pnpm/vite@4.5.0/node_modules/vite/dist/node/chunks/dep-bb8a8339.js:28725:11)
    at resolvePackageEntry (file:///Users/bruno/git/spotlight/node_modules/.pnpm/vite@4.5.0/node_modules/vite/dist/node/chunks/dep-bb8a8339.js:28722:5)
    at tryNodeResolve (file:///Users/bruno/git/spotlight/node_modules/.pnpm/vite@4.5.0/node_modules/vite/dist/node/chunks/dep-bb8a8339.js:28453:20)
    at Context.resolveId (file:///Users/bruno/git/spotlight/node_modules/.pnpm/vite@4.5.0/node_modules/vite/dist/node/chunks/dep-bb8a8339.js:28212:28)
    at Object.resolveId (file:///Users/bruno/git/spotlight/node_modules/.pnpm/vite@4.5.0/node_modules/vite/dist/node/chunks/dep-bb8a8339.js:44276:64)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async TransformContext.resolve (file:///Users/bruno/git/spotlight/node_modules/.pnpm/vite@4.5.0/node_modules/vite/dist/node/chunks/dep-bb8a8339.js:43992:23)
    at async normalizeUrl (file:///Users/bruno/git/spotlight/node_modules/.pnpm/vite@4.5.0/node_modules/vite/dist/node/chunks/dep-bb8a8339.js:41836:34)
    at async file:///Users/bruno/git/spotlight/node_modules/.pnpm/vite@4.5.0/node_modules/vite/dist/node/chunks/dep-bb8a8339.js:41998:47
    at async Promise.all (index 7)

/Users/bruno/git/spotlight/demos/astro-playground:
 ERR_PNPM_RECURSIVE_RUN_FIRST_FAIL  @spotlightjs/astro-playground@0.0.0 dev: `astro dev`
Exit status 1
 ELIFECYCLE  Command failed with exit code 1.
bruno@DHPKTVF22X spotlight %
```

--- a/packages/website/src/content/docs/contribute/index.mdx
+++ b/packages/website/src/content/docs/contribute/index.mdx
@@ -17,10 +17,16 @@ Ensure the node and pnpm are available:
 volta install node pnpm
 ```
 
-Lastly, install the package requirements:
+Install the package requirements:
 
 ```shell
 pnpm install
+```
+
+And finally, run an initial build:
+
+```shell
+pnpm build
 ```
 
 ## Development
@@ -57,11 +63,11 @@ pnpm link --global @spotlightjs/spotlight
 
 ## Repository Structure
 
-The monorepo is mostly straight forward, but here's a quick summary: 
+The monorepo is mostly straight forward, but here's a quick summary:
 
 ```
 spotlight
-└── packages                
+└── packages
     ├── overlay      // @spotlightjs/overlay   - the main overlay
     ├── sidecar      // @spotlightjs/sidecar   - the sidecar (proxy) for streaming data
     ├── spotlight    // @spotlightjs/spotlight - the main spotlight package, combining overlay and sidecar

